conf/messages

```diff
@@ -58,6 +58,7 @@ vatDetails.vatOptOut.info = You cannot opt out if your taxable turnover has been
 vatDetails.vatOptOut.pending = Your request to opt out is being processed.
 vatDetails.mtdSignup.link = Sign up for Making Tax Digital for VAT
 vatDetails.mtdSignup.info = If your taxable turnover exceeds the VAT threshold, you must sign up to Making Tax Digital for VAT.
+vatDetails.mtdSignup.info.newStatuses = You must sign up to Making Tax Digital for VAT if you''re not exempt from VAT, and your taxable turnover exceeds the £85,000 threshold.
```

Comment:
As mentioned above, as the feature switch is not needed you can just update the original message with this content. The welsh is okay to be left as is and will be played in a separate story.